### PR TITLE
Document Candy Machine v3 (Part 3)

### PR DESCRIPTION
**Pull request by lorisleiva**
- Write "[Inserting Items](https://docs-git-loris-cmv3-part3.metaplex.dev/programs/candy-machine/inserting-items)"
- Write "[Candy Guards](https://docs-git-loris-cmv3-part3.metaplex.dev/programs/candy-machine/candy-guards)"

### CODE REVIEW

File: docs/01-programs/02-candy-machine/04-inserting-items.md [hunk 1]
```diff
@@ -2,15 +2,200 @@
...
+
+</div>
+</AccordionItem>
+</Accordion>
+
+## Inserting Items
+
+Now that we have a name and URI for all of our items, all we need to do is insert them into our Candy Machine account.
+
+Note that, because transactions are limited to a certain size, we cannot insert thousands of items within the same transaction. The number of items you can insert per transaction will depend on the **Name Length** and **URI Length** attributes defined in your Config Line Settings. The shorter your names and URIs are, the more you’ll be able to fit into a transaction.
```
Comment (febo) [inline docs/01-programs/02-candy-machine/04-inserting-items.md:74]: Perhaps we need to make it more explicit the effect of `Name Length` and `URI Length` values to the length of the `Name` and `URI` – i.e., the length of `Name` and `URI` are constraint to the `Name Length` and `URI Length` values.

Comment (febo) [reply docs/01-programs/02-candy-machine/04-inserting-items.md:74]: Maybe this need to be on the `Inserting Items Using Prefixes` section.

File: docs/01-programs/02-candy-machine/04-inserting-items.md [hunk 2]
```diff
@@ -2,15 +2,200 @@
...
 
-In the meantime, you can learn more about Candy Machine V3 via the following resources:
+## Conclusion
 
-- [Candy Machine Core Program documentation](https://github.com/metaplex-foundation/metaplex-program-library/tree/master/candy-machine-core/program)
-- [Candy Guard Program documentation](https://github.com/metaplex-foundation/mpl-candy-guard)
+And just like that, we have a loaded Candy Machine ready to mint NFTs! However, we've not created any requirements for our minting process. How can we configure the price of the mint? How can we ensure that buyers are holders of a specific token or an NFT from a specific collection? How can we set the start date of our mint? What about the end conditions?
 
-Thank you!
+[On the next page](/programs/candy-machine/candy-guards), we’ll talk about Candy Guards which make all of this possible.
```
Comment (febo) [inline docs/01-programs/02-candy-machine/04-inserting-items.md:201]: We should add a note to say that minting is only allowed when all config lines have been added – we added a specific check on CMv3 for that.

**Comment by febo** [comment]
The diagrams are super nice!

File: docs/01-programs/02-candy-machine/04-inserting-items.md [hunk 3]
```diff
@@ -2,15 +2,200 @@
...
+
+</div>
+</AccordionItem>
+</Accordion>
+
+## Overriding Existing Items
+
+When inserting items, you may provide the position in which these items should be inserted. This enables you to insert items in any order you want but also allows you to update items that have already been inserted.
+
+Note that, once minting has started, items can no longer be changed.
```
[the remark this answers is not in the document]
Comment (lorisleiva) [reply docs/01-programs/02-candy-machine/04-inserting-items.md:161]: Oh really?! I didn't know sorry, I'll remove that sentence.

File: docs/01-programs/02-candy-machine/04-inserting-items.md (hunk 2 above)
Comment (lorisleiva) [reply docs/01-programs/02-candy-machine/04-inserting-items.md:201]: Good shout!

File: docs/01-programs/02-candy-machine/04-inserting-items.md (hunk 1 above)
Comment (lorisleiva) [reply docs/01-programs/02-candy-machine/04-inserting-items.md:74]: Good point, I'll add that info somewhere.

**Review by febo — APPROVED** [review]

**Pull request merged**